sdk/servicebus/azservicebus/client.go

```diff
@@ -24,23 +24,23 @@ type ServiceBusClient struct {
 
 // ServiceBusClientOption is the type for an option that can configure ServiceBusClient.
 // For an example option, see `ServiceBusWithConnectionString`
-type ServiceBusClientOption func(client *ServiceBusClient) error
+type ServiceBusClientOption func(client *Client) error
```

Comment:
Shouldn't these then remove ServiceBus prefix as well, ie `ClientOptions`?

Reply:
Indeed and renamed.